### ISSUE TITLE
Noticing DNS requests from client that hasn't been setup to sync and where telemetry is off

### ISSUE THREAD

**Issue by dwarf-king-hreidmar** [streetwriters/notesnook#1693]
I installed notesnook because I love the UI and I'm really excited to see the progress on a self hosted sync server / container image. However, I noticed a ton of traffic to api.notesnook.com and events.streetwriters.co even with telemetry turned off and without ever signing into an account. Is this expected? What are these calls for? 

Thanks so much for all your work so far.

**Comment by thecodrr** [comment]
>  I noticed a ton of traffic to api.notesnook.com and events.streetwriters.co

Describe ton. I am noticing only 2 requests.

1. `https://api.notesnook.com/announcements/active`

This fetches the announcements on app startup. Announcements show up at the very top of the list which includes stuff like offers, maintenance forewarning etc.

2. `https://events.streetwriters.co/sse`

This is not a request but an event stream. It is only connected to after you log in so not sure what you are seeing.

You can analyze all the outgoing requests yourself to see if they contain telemetry.

**Comment by dwarf-king-hreidmar** [comment]
Thanks for your response. Tons was an overstatement it turns out. I was not properly filtering my logs and I'm sorry for that. I have telemetry disabled on the client. I also don't have an account with notesnook to try to sign on so I didn't expect to see outbound traffic. Thanks for the clarification above. I'm seeing the following

Seeing 4 requests to analytics.streetwriters.co on startup even with telemetry turned off.
Seeing 4 requests to api.notesnook.com. 

About the events.streetwritters.co I only saw this earlier in my testing. Here is a sample. But I assure you I don't have an account to log in with: 

```
Jan xx xx:59:33 dnsmasq[53646]: query[AAAA] [events.streetwriters.co](http://events.streetwriters.co/) from xxxxx
Jan xx xx:59:33 dnsmasq[53646]: query[A] [events.streetwriters.co](http://events.streetwriters.co/) from xxxxxxxx
Jan xx xx:59:33 dnsmasq[53646]: query[A] [events.streetwriters.co](http://events.streetwriters.co/) from xxxx
Jan xx xx:59:33 dnsmasq[53646]: query[AAAA] [events.streetwriters.co](http://events.streetwriters.co/) from xxxx

```

**Comment by thecodrr** [comment]
> 4 requests to analytics.streetwriters.co

This is probably due to loading the analytics script. I will see if there's a way to turn that off when telemetry is disabled (there should be).

> 4 requests to api.notesnook.com

Can you share the full details of these requests?

> About the events.streetwriters.co I only saw this earlier in my testing. Here is a sample. But I assure you I don't have an account to log in with.

I will have to investigate more. My guess is that you were previous logged in & then you logged out but it didn't clear the tokens properly. Can you try in an incognito window?

**Comment by dwarf-king-hreidmar** [comment]
Is there an incognito window on the desktop app?

Happy to grab more details. Where would the desktop app drop detailed logs about the calls?  I’m just seeing these on my DNS server.

**Comment by thecodrr** [comment]
Use `Ctrl + Shift + I` in the desktop app & then switch to the Network tab.

**Comment by dwarf-king-hreidmar** [comment]
Thanks for the tip!

For https://api.notesnook.com/announcements/active
Request Method: GET

These were just pulling updates as you noted above.


Foranalytics.streetwriters.co/api/collect
Request Method: Put

Payload: 

```{
  "website": "xxxx-xxxxx-xxxx-xxxx-",
  "hostname": "app.notesnook.com",
  "screen": "1920x1080",
  "language": "en-US",
  "cache": null,
  "url": "/",
  "referrer": ""
}```


Another put had: 
```
{
  "website": "xxxx-xxx-xxx-xxx",
  "hostname": "app.notesnook.com",
  "screen": "1920x1080",
  "language": "en-US",
  "cache": null,
  "event_type": "version",
  "event_value": "xxxxx-desktop-linux",
  "url": "/notebooks/xxxx/xxxxxxx"
}```


Thanks for removing these when you're offline. Will these also be absent when there is an option to choose our own sync server?

**Comment by dwarf-king-hreidmar** [comment]
You might also like to know that the second put with the second payload always returns a 500 Internal Server Error

**Comment by thecodrr** [comment]
Both of these have been fixed in #1718
